common/src/stack/command/stack/commands/report/host/interface/imp_redhat.py

```diff
@@ -33,6 +34,9 @@ def run(self, args):
 			vlanid	= row['vlan']
 			default = row['default']
 
+			if ip and not netname:
+				raise CommandError(self, f'interface "{interface}" on host "{row["host"]}" has an IP but no network')
```

Comment:
AIUU, `CommandError()` in implementations and plugins need a reference to the command object as the first argument, so `self.owner`.  Can you verify you're not having a problem here (and in the below implementation as well)?

Reply:
Error appears as:
```
sd-stacki-010:~/stacki # stack report host interface sd-stacki-009
error - interface "eth4" on host "sd-stacki-009" has an IP but no network
```
Seems fine.